src/lib/db.js

```diff
@@ -26,7 +26,8 @@ function executeSQLPlus (sqlFilepath) {
     const host = connection.host
     const port = connection.port
     const database = connection.database
-    const login = user + '/' + password + '@' + host + ':' + port
+    // TODO: Oracle connection string passed as configuration to ledger
+    const login = user + '/' + password + '@' + host + '/xe:' + port
```

Comment:
This conflicts with the documentation: https://docs.oracle.com/cd/B19306_01/server.102/b14357/qstart.htm#i1055567